### ISSUE TITLE
Fails to read profile file if path has space_char (ASCII 32)

### ISSUE THREAD

**Issue by singh-abhijeet** [jiffyclub/snakeviz#120]
`snakeviz` is unable to read profile file if path (to profile file) has space character (alt+32) and throws the following error:

```python
ERROR:tornado.application:Uncaught exception GET /snakeviz/%2Fhome%2Fabhijeet%2FDocuments%2FSBH%2FDir+with+spaces%2Fprofile.prof (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:8080', method='GET', uri='/snakeviz/%2Fhome%2Fabhijeet%2FDocuments%2FSBH%2FDir+with+spaces%2Fprofile.prof', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/home/abhijeet/anaconda3/lib/python3.6/site-packages/snakeviz/main.py", line 31, in get
    s = Stats(profile_name)
  File "/home/abhijeet/anaconda3/lib/python3.6/pstats.py", line 72, in __init__
    self.init(arg)
  File "/home/abhijeet/anaconda3/lib/python3.6/pstats.py", line 86, in init
    self.load_stats(arg)
  File "/home/abhijeet/anaconda3/lib/python3.6/pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/abhijeet/Documents/SBH/Dir+with+spaces/profile.prof'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/abhijeet/anaconda3/lib/python3.6/site-packages/tornado/web.py", line 1541, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/abhijeet/anaconda3/lib/python3.6/site-packages/snakeviz/main.py", line 33, in get
    raise RuntimeError('Could not read %s.' % profile_name)
RuntimeError: Could not read /home/abhijeet/Documents/SBH/Dir+with+spaces/profile.prof.
ERROR:tornado.access:500 GET /snakeviz/%2Fhome%2Fabhijeet%2FDocuments%2FSBH%2FDir+with+spaces%2Fprofile.prof (127.0.0.1) 3.75ms
```

**Details**
OS: Linux 4.15.0-38-generic #41-Ubuntu
pwd: `/home/abhijeet/Documents/SBH/Dir with spaces`
Profile file path/location: `/home/abhijeet/Documents/SBH/Dir with spaces/profile.prof`
Command: `snakeviz profile.prof`

**Comment by jiffyclub** [comment]
Thanks for the report! In my local testing this was addressed by 9505c88, please give master a try and let me know if it doesn't work for you.

**Comment by singh-abhijeet** [comment]
Thanks for addressing this @jiffyclub.
